x/staking/keeper/validator_test.go

```diff
@@ -292,146 +292,6 @@ func TestValidatorBasics(t *testing.T) {
 	require.False(t, found)
 }
 
-// test how the validators are sorted, tests GetBondedValidatorsByPower
-func GetValidatorSortingUnmixed(t *testing.T) {
```

Comment:
Why was this test deleted? These are vital tests of functionality which must be kept unless this is clearly duplicated somewhere (blocking comment)

Reply:
i got this as unused code and couldn't find its uses used in the codebase, do we want to keep unused code around? I'm fine with putting it back

Reply:
We definitely want both these tests which have been deleted in this file

Reply:
just for the heads up these tests fail and arent currently run, will comment them out and put them back in

Reply:
@marko, let's not comment them out, but rather at `t.SkipNow()` at the very top of the test function. Also, let's open up an issue and reference it to be fixed asap.

i.e.

```go
func GetValidatorSortingUnmixed(t *testing.T) {
    // TODO: Evaluate and fix broken unit test.
    // ref: link
    t.SkipNow()

    // ...
}
```

Reply:
I second @alexanderbez

Reply:
added it in